compose/src/main/java/com/stripe/android/compose/FormFragment.kt

```diff
@@ -0,0 +1,42 @@
+package com.stripe.android.compose
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.stripe.android.compose.forms.SofortForm
+import com.stripe.android.compose.forms.SofortFormViewModel
+
+class FormFragment : Fragment() {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View = ComposeView(inflater.context).apply {
+        layoutParams = ViewGroup.LayoutParams(
+            ViewGroup.LayoutParams.MATCH_PARENT,
+            ViewGroup.LayoutParams.MATCH_PARENT
+        )
+
+        setContent {
+            StripeTheme {
+                Column(Modifier.fillMaxSize()) {
+                    val sofortFormViewModel = viewModel<SofortFormViewModel>()
+                    val paramStr by sofortFormViewModel.params.observeAsState("")
```

Comment:
What does the argument to `observeAsState` represent?

Reply:
The initial value

Reply:
If the type of `params` is `LiveData<PaymentMethodCreateParams?>` I would expect the initial value to be of that type. Am I missing something?

Reply:
No, I set it to null, I am surprised that worked at all.